src/main/java/org/jitsi/jibri/CallUrlInfo.kt

```diff
@@ -0,0 +1,16 @@
+package org.jitsi.jibri
+
+/**
+ * We assume the 'baseUrl' represents a sort of landing page (on the same
+ * domain) where we can set the necessary local storage values.  The call
+ * url will be created by joining [baseUrl] and [callName] with a "/"
+ */
+data class CallUrlInfo(
+        val baseUrl: String = "",
+        val callName: String = ""
+) {
+
+    fun getCallUrl(): String {
+        return baseUrl + "/" + callName
```

Comment:
Probs more idiomatic to use a property here instead 

`val = callUrl = "$baseUrl/$callName"`

Reply:
cool, fixed that